sam-styles/packages/components/button/styles/button.scss

```diff
@@ -831,4 +831,57 @@ $button-shadow: 0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2);
     @include u-flex("column");
     flex-grow: 1;
   }
-}
\ No newline at end of file
+}
+
+.usa-button-pill {
+  @include u-radius("pill");
+  @include u-display("flex");
+  @include u-flex('align-center'); // Vertically align items in the center
+  @include u-flex('justify-center'); // Horizontally center content
+
+  svg {
+    width: 16px !important;
+    height: 16px !important;
+    fill: currentColor !important;
+    stroke-width: 1px !important;
+    stroke: color("secondary-darker");
+  }
+
+  &--next {
+    svg {
+      @include u-margin-left(1);
+      @include u-margin-top(1px);
+    }
+    &.usa-button:active svg,
+    &.usa-button:disabled svg,
+    &.usa-button--active svg {
+      stroke: color("white");
+    }
+  }
+
+  &--previous {
+    svg {
+      @include u-margin-right(1);
+      @include u-margin-top(1px);
+    }
+    &.usa-button--active svg {
+      stroke: color("secondary-darker");
+    }
+    &:disabled svg {
+      stroke: color("base");
+    }
+  }
+
+  &--cancel {
+    svg {
+      @include u-margin-right(1);
+      @include u-margin-top(1px);
```

Comment:
If you keep the width/height of 14px, up this to 2px.

1px:
<img width="1109" alt="Screenshot 2023-08-25 at 9 24 26 AM" src="https://github.com/GSA/sam-styles/assets/72805180/895108a0-b1e9-454e-a0f8-a3e902cb7399">

2px:
<img width="1109" alt="Screenshot 2023-08-25 at 9 24 31 AM" src="https://github.com/GSA/sam-styles/assets/72805180/4b65fecf-abcd-400b-ae83-a33c74c2b093">


And if you do revert t
<img width="1109" alt="Screenshot 2023-08-25 at 9 24 39 AM" src="https://github.com/GSA/sam-styles/assets/72805180/e5e1a667-f292-46cd-8245-dc70d41ac01d">
he height/width to 16px, set this to 3px:

Reply:
kept the same size and increased margin to 2 px.